EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,77 @@
+---
+eip: eip-xxxx
+title: GETCONTRACT opcode
+description: Global byte code accessing by it's hash
+author: Tim Pechersky (@peersky)
+discussions-to: https://ethereum-magicians.org/t/erc-7744-code-index/20569
```

Comment:
I recommend creating a discussions topic specific to this EIP rather than linking to the ERC discussion.  You can add a link to the ERC discussion in the EIP discussion.